conf/cron_kanboard

```diff
@@ -0,0 +1,2 @@
+# Execute the daily cronjob at 8am
+0 8 * * * cd "__FINALPATH__" && ./cli cronjob >/dev/null 2>&1
```

Comment:
Shouldn't we ensure we're using the PHP version defined for this package?

Reply:
@JimboJoe __PHPVERSION__ does not seem to be replaced by the correct PHP version number ... Any hints to correctly set this PHP version?

`0 8 * * * __APP__ cd "__FINALPATH__" && PHP__PHPVERSION__ ./cli cronjob >/dev/null 2>&1`

Reply:
Well..., I just added a `ynh_replace_sting` in the install and upgrade script...

`ynh_replace_string --match_string="__PHPVERSION__" --replace_string="$phpversion" --target_file="/etc/cron.d/$app"`